### PR TITLE
Addition of the liveness endpoint and `addTo` function 

### PR DESCRIPTION
**Pull request by KevinJCross**
Work in progress

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,17 +28,12 @@ app.get("/status", healthcheck.configure({
...
 
 ## Publishing
 
-To package and publish use the Makefile target. This requires your artifactory API key to be set to an
-environment variable called `JFROG_API_KEY`. You can get your key from [your artifactory profile](https://artifactory.reform.hmcts.net/artifactory/webapp/#/profile).
-
-```
-JFROG_API_KEY=fake-key make publish
-```
+We use Travis for publishing via build pipelines once code has been merged.
```
Comment (timja) [inline README.md]:
```suggestion
Bump the version (SemVer) and create a release in the GitHub UI, Travis CI will then build test and release to the npm registry.
```

File: healthcheck/install.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const routes = require('./routes')
+const outputs = require('./outputs')
+
+function addTo(app, config) {
+  console.log("adding healthCheck ")
```
Comment (timja) [inline healthcheck/install.js]: can we remove these debug console.logs, no one will want these in their app

File: healthcheck/routes.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 'use strict'
-const checks = require('./checks'),
-      outputs = require('./outputs'),
-      versionFile = require('./versionFile');
+const checks = require('./checks')
+const outputs = require('./outputs')
```
Comment (timja) [inline healthcheck/routes.js]: can you separate these formatting changes to a different PR please

File: healthcheck/install.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  app.get('/health', routes.configure(config))
+  console.log("adding liveness")
+  app.get('/health/liveness', (req, res) => res.status(200).json(outputs.status(outputs.UP)))
```
Comment (timja) [inline healthcheck/install.js]: isn't 200 the default?

Comment (timja) [inline healthcheck/install.js]: is there a test for this? (I couldn't see one)

Comment (klawani) [reply healthcheck/install.js]: just added the file installTest.js

File: test/unit/installTest.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const request = require('supertest');
```
Comment (timja) [inline test/unit/installTest.js]: code style seems different in this file, the other file you were changing isn't using semi colons

File: healthcheck/install.js (hunk 4 above)
Comment (klawani) [reply healthcheck/install.js]: what do you mean?

Comment (timja) [reply healthcheck/install.js]: this?
```suggestion
  app.get('/health/liveness', (req, res) => res.json(outputs.status(outputs.UP)))
```

File: healthcheck/routes.js (hunk 3 above)
Comment (klawani) [reply healthcheck/routes.js]: done.

File: test/unit/installTest.js (hunk 5 above)
Comment (klawani) [reply test/unit/installTest.js]: that is true. I changed it to use semi colons to be consistent with other test files that were there :-)

Comment (timja) [reply test/unit/installTest.js]: hah fine, whole repo should get a once over separately then
